repository/src/main/java/org/pentaho/platform/repository/hibernate/usertypes/EmptyStringUserType.java

```diff
@@ -24,8 +24,11 @@
 import org.apache.commons.logging.LogFactory;
 import org.hibernate.Hibernate;
 import org.hibernate.HibernateException;
+import org.hibernate.engine.spi.SharedSessionContractImplementor;
+import org.hibernate.type.StandardBasicTypes;
 import org.hibernate.usertype.UserType;
-import org.hibernate.util.EqualsHelper;
+//EqualsHelper removed after hibernate-core-5.3.1.Final.jar
+//import org.hibernate.util.EqualsHelper;
```

Comment:
Is this comment necessary?  The block below seems to document everything well enough.